data/translations/bn/state-data.json

```diff
@@ -3,311 +3,407 @@
-        "ip_deadline": "নির্বাচনের আগে চতুর্থ বুধবার",
-        "online_deadline": "নির্বাচনের আগে চতুর্থ বুধবার",
-        "mail_postmarked_deadline": "নির্বাচনের আগে চতুর্থ বুধবারের মধ্যে পোস্টমার্ক করতে হবে",
-        "confirm_registration_link": "https://s1.sos.mo.gov/elections/goVoteMissouri/register.aspx"
+        "ip_deadline": "বুধবার, ১২ অক্টোবর, ২০২২",
+        "online_deadline": "বুধবার, ১২ অক্টোবর, ২০২২",
+        "mail_postmarked_deadline": "পোস্ট চিহ্ন করা আবশ্যক (বুধবার, ১২ অক্টোবর, ২০২২)",
+        "confirm_registration_link": "https://s1.sos.mo.gov/elections/goVoteMissouri/register.aspx",
+        "default_ip_deadline": "নির্বাচনের আগে চতুর্থ বুধবার",
+        "default_online_deadline": "নির্বাচনের আগে চতুর্থ বুধবার",
+        "default_mail_postmarked_deadline": "নির্বাচনের আগে চতুর্থ বুধবারের মধ্যে পোস্টমার্ক করতে হবে"
     },
     "mt": {
         "hp_link": "https://sosmt.gov/elections/",
         "more_info_link": "https://sosmt.gov/elections/vote/",
-        "ip_deadline": "নির্বাচনের দিন পর্যন্ত এবং সহ উপলব্ধ",
-        "mail_postmarked_deadline": "নির্বাচনী দিনের ৩০ দিন আগে পোস্টমার্ক করা আবশ্যক",
-        "confirm_registration_link": "https://sosmt.gov/elections/vote/"
+        "ip_deadline": "সোমবার, ৭ নভেম্বর, ২০২২",
+        "mail_postmarked_deadline": "পোস্ট চিহ্ন করা আবশ্যক (মঙ্গলবার, ১১ অক্টোবর, ২০২২)",
+        "confirm_registration_link": "https://sosmt.gov/elections/vote/",
+        "default_ip_deadline": "নির্বাচনের দিন পর্যন্ত এবং সহ উপলব্ধ",
+        "default_mail_postmarked_deadline": "নির্বাচনী দিনের ৩০ দিন আগে পোস্টমার্ক করা আবশ্যক"
     },
     "ne": {
         "hp_link": "https://sos.nebraska.gov/elections/2020-elections",
         "registration_link": "https://www.nebraska.gov/apps-sos-voter-registration/",
         "more_info_link": "https://sos.nebraska.gov/elections/voter-forms",
-        "ip_deadline": "নির্বাচনের আগের দ্বিতীয় শুক্রবার",
-        "online_deadline": "নির্বাচনের আগের তৃতীয় শুক্রবার",
-        "mail_postmarked_deadline": "নির্বাচনের দিন আগে তৃতীয় শুক্রবারের মধ্যে পোস্টমার্ক করতে হবে",
-        "confirm_registration_link": "https://www.votercheck.necvr.ne.gov/voterview"
+        "ip_deadline": "শুক্রবার, ২৮ অক্টোবর, ২০২২",
+        "online_deadline": "শুক্রবার, ২১ অক্টোবর, ২০২২",
+        "mail_postmarked_deadline": "পোস্ট চিহ্ন করা আবশ্যক (শুক্রবার, ২১ অক্টোবর, ২০২২)",
+        "confirm_registration_link": "https://www.votercheck.necvr.ne.gov/voterview",
+        "default_online_deadline": "নির্বাচনের আগের তৃতীয় শুক্রবার",
+        "default_mail_postmarked_deadline": "নির্বাচনের দিন আগে তৃতীয় শুক্রবারের মধ্যে পোস্টমার্ক করতে হবে"
     },
     "nv": {
         "hp_link": "https://www.nvsos.gov/sos/elections",
         "registration_link": "https://www.nvsos.gov/SOSVoterServices/start.aspx",
         "more_info_link": "https://www.nvsos.gov/sos/elections/voters/registering-to-vote",
-        "ip_deadline": "নির্বাচনের আগে চতুর্থ মঙ্গলবার",
```

Comment:
We need to keep the current `ip_deadline` as a `default_` value for fallback purposes.

Reply:
fixed